main.go

```diff
@@ -472,13 +472,17 @@ func aggregateTimemap(urir string, dttmp *time.Time, sess *Session) (basetm *lis
 }
 
 func parseURI(uri string) (urir string, err error) {
-	if !regs["isprtcl"].MatchString(uri) {
-		uri = "http://" + uri
+	uescd, err := url.QueryUnescape(uri)
+	if !regs["isprtcl"].MatchString(uescd) {
+		uescd = "http://" + uri
```

Comment:
Here, the protocol is prefixed to the original `uri`, not the unescaped one, so the behavior will be different for the escaped URIs with and without scheme. That said, if we were not to introduce the new variable, this problem will not arise in the first place.

Reply:
Good catch here! `uri` ought to be `uescd`, which makes the introduction of the additional var moot.